brick/main.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	var a, b float64
+	fmt.Println("Enter the hole dimensions (2 numbers):")
+	fmt.Scan(&a, &b)
+	var p, q, r float64
+	fmt.Println("Enter the brick dimensions (3 numbers):")
+	fmt.Scan(&p, &q, &r)
+	if a >= p && b >= q || b >= p && a >= q || a >= q && b >= r || a >= r && b >= q || a >= p && b >= r || a >= r && b >= p {
```

Comment:
Even if this condition is correct, it's very hard to understand it. What about trying to sort `p, q, r` and `a, b`. Would it help to solve it any nicer?